x-pack/legacy/plugins/lens/public/editor_frame_plugin/embeddable/embeddable.tsx

```diff
@@ -122,6 +122,7 @@ export class Embeddable extends AbstractEmbeddable<LensEmbeddableInput, LensEmbe
 
   destroy() {
     super.destroy();
+
     if (this.domNode) {
       unmountComponentAtNode(this.domNode);
     }
```

Comment:
This looks like it'll cause a build failure because the domNode is no longer being stored on `this`. Might also cause a test failure